sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosClientBuilder.java

```diff
@@ -772,11 +797,30 @@ boolean isReadRequestsFallbackEnabled() {
      * Returns the client telemetry config instance for this builder
      * @return the client telemetry config instance for this builder
      */
-    public CosmosClientTelemetryConfig clientTelemetryConfig() {
-        return ImplementationBridgeHelpers
+    CosmosClientTelemetryConfig getClientTelemetryConfig() {
+        return this.clientTelemetryConfig;
+    }
+
+    /**
+     * Returns the client telemetry config instance for this builder
```

Comment:
The doc is not correct anymore, since now we return the CosmosClientBuilder instance.

Reply:
Fixed